### DISCUSSION TITLE
email migration from Plesk panel to cPanel

### DISCUSSION THREAD

**Discussion by dokoyahaya47** [imapsync/imapsync#303] [Q&A] [answered]
good day house.. please i want a guild on how to transfer email accounts and their content from plesk panel to cpanel.

**Comment by gilleslamiral** [comment, marked as answer]
imapsync usage is the same for any imap account

**Comment by David-Garcia9919** [comment]
If you’re moving email accounts from Plesk to cPanel, the process usually involves recreating the same mailboxes on the destination server and then transferring the mailbox data using an IMAP-based method. Since both panels support standard IMAP access, the migration is mostly about connecting to the old server and syncing everything to the new one.

In my case, I recreated the mailboxes in cPanel and then used an IMAP to IMAP migration tool to copy the mail folders and message content. I’ve used the EdbMails [IMAP Migration Tool](https://www.edbmails.com/pages/imap-migration-tool.html), which has an IMAP to IMAP option that lets you connect to the Plesk mail server on one side and the cPanel server on the other. It simply transfers the mailbox data without altering the folder structure, so the accounts look the same after the move.

You can follow the same approach with any IMAP-based method:

1. Create the same email accounts in cPanel.
2. Confirm IMAP access and login details for both servers.
3. Migrate emails from source (Plesk) to destination (cPanel).

This avoids manual downloads, and it works well even when the accounts are large or users have many folders. If you need to migrate multiple accounts, doing it through an IMAP tool can save a lot of time and reduce the chance of missing data.

Detailed Steps by Step Guide:
https://www.edbmails.com/pages/imap-to-imap-server-migration-steps.html
